source/geometries/Next100FieldCage.cc

```diff
@@ -51,17 +52,21 @@ Next100FieldCage::Next100FieldCage(G4double grid_thickn):
   n_panels_            (18),
   active_ext_radius_   (active_diam_/2. / cos(pi/n_panels_)),
 
+  grid_thickn_         (grid_thickn),
+
   cathode_int_diam_    (965. * mm),
   cathode_ext_diam_    (1020.* mm),
   cathode_thickn_      (13.5  * mm),
-  grid_thickn_         (grid_thickn),
+  cathode_mesh_diam_   (5.0*mm),
 
   //  teflon_drift_length_ (1178.*mm), //distance from the gate to the beginning of the cathode volume
   teflon_long_length_   (465. * mm),
   teflon_buffer_length_ (241. * mm),
   teflon_thickn_       (5. * mm),
 
   el_gap_length_ (10. * mm),
+  el_mesh_diam_  (2.5 * mm),
+  el_mesh_rot_   (15 * deg),
```

Comment:
Are the meshes installed at LSC rotated by exactly this angle? Was it measured? Or is this angle fixed by construction?